### DISCUSSION TITLE
waybar hints have ugly background

### DISCUSSION THREAD

**Discussion by Gigas002** [prasanthrangan/hyprdots#1001] [Q&A] [answered]
As the title says, here it is:

![image](https://github.com/prasanthrangan/hyprdots/assets/24297712/50f2862c-ae66-4947-968f-110dc19c4fe8)

I wonder what can be the cause of such vulgar behavior

**Comment by kRHYME7** [comment, marked as answer]
@Gigas002 Reported this multiple times, maintainer said it's something to do with the gtk3 being used. (I assumed the one in the ~/.themes) 

https://github.com/prasanthrangan/hyprdots/issues/686
https://github.com/prasanthrangan/hyprdots/issues/190
